extensions/amp-analytics/0.1/vendors/mapp_intelligence.json

```diff
@@ -0,0 +1,39 @@
+{
+  "vars": {
+    "actionName": "webtrekk_ignore",
+    "contentId": "${title}",
+    "mediaName": "${id}",
+    "everId": "${clientId(amp-wt3-eid)}"
+  },
+  "requests": {
+    "trackURL": "https://${trackDomain}/${trackId}/wt",
+    "basePrefix": "?p=440,${contentId},1,${screenWidth}x${screenHeight},${screenColorDepth},1,",
+    "baseSuffix": ",${documentReferrer},${viewportWidth}x${viewportHeight},0&tz=${timezone}&eid=${everId}&la=${browserLanguage}",
+    "parameterPrefix": "${basePrefix}${timestamp}${baseSuffix}",
+    "parameterSuffix": "&pu=${sourceUrl}&eor=1",
+    "pageview": "${trackURL}${parameterPrefix}&${extraUrlParams}&cp570=${pageLoadTime}${parameterSuffix}",
+    "event": "${trackURL}${parameterPrefix}&ct=${actionName}&${extraUrlParams}${parameterSuffix}",
+    "scroll": "${trackURL}${parameterPrefix}&ct=${actionName}&ck540=${verticalScrollBoundary}${parameterSuffix}",
+    "mediaPrefix": "${trackURL}${basePrefix}${baseSuffix}&mi=${mediaName}",
+    "mediaSuffix": "&mt1=${currentTime}&mt2=${duration}&${extraUrlParams}${parameterSuffix}&x=${playedTotal}",
+    "mediaPlay": "${mediaPrefix}&mk=play${mediaSuffix}",
+    "mediaPause": "${mediaPrefix}&mk=pause${mediaSuffix}",
+    "mediaPosition": "${mediaPrefix}&mk=pos${mediaSuffix}",
+    "mediaEnded": "${mediaPrefix}&mk=eof${mediaSuffix}"
+  },
+  "extraUrlParamsReplaceMap": {
+    "pageParameter": "cp",
+    "contentGroup": "cg",
+    "actionParameter": "ck",
+    "sessionParameter": "cs",
+    "ecommerceParameter": "cb",
+    "urmCategory": "uc",
+    "campaignParameter": "cc",
+    "mediaCategory": "mg"
+  },
+  "transport": {
```

Comment:
To confirm, the server only supports GET requests?

Reply:
Hi @zhouyx,

yes, only get requests are supported.
I also added Mapp Intelligence to the vendor list in commit 44ec184becc28dee8adadb2641cc2e16ebeb3af6.

I also have a question about the Google Tag Manager AMP implementation (https://support.google.com/tagmanager/answer/9205783?hl=en&ref_topic=3002579) - will our pull request here be enough to be included to Google Tag Manager or is there some other process in order to get our Tag running in GTM?

Thank you very much for you support.

Reply:
Thanks a lot @avimehta. Since we undergone a company and brand change it would be nice to have that reflected in the vendor list as well (once the commit is approved). It would also be nice to offer it as a template in the community gallery in GTM - can we add that ourselves or is this also done by the GTM team? 

Best regards,

Stella